commands/root.go

```diff
@@ -90,3 +107,28 @@ func Execute() {
 		os.Exit(1)
 	}
 }
+
+func getCommitAndDirty() (commit, dirty string) {
+	var d, c string
+
+	info, ok := debug.ReadBuildInfo()
+
+	if !ok {
```

Comment:
if you can't read the build infos:
- you should not continue reading the returned value, that could crash. Instead, return an error or a valid value.
- you should not print anything